public/Lab-Files/Lesson02/Challenges/text-level-elements.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Text-Elements Challenge</title>
+</head>
+<body>
+    <p><strong>Web development</strong> is a broad term for the work involved in developing a web site for the<br>
+    <em>Internet</em> (World Wide Web) or an <em>intranet</em> (a private network). Web development can range <br>
+    from developing the simplest static single page of plain text to the most complex web-based <br>
+    internet applications, electronic businesses, and social network services. A more <br>
+    comprehensive list of tasks to which web development commonly refers, may include web <br>
+    engineering, web design, web content development, client liaison, client-side/server-side <br>
+    scripting, web server and network security configuration, and e-commerce development. Among <br>
+    web professionals, "<em>web development</em>" usually refers to the main non-design aspects of <br>
+    building web sites: <strong>writing markup and coding</strong>. Most recently Web development has come to <br>
+    mean the creation of content management systems or <strong>CMS</strong>. These CMS can be made from scratch,<br>
+    proprietary or open source. In broad terms the CMS acts as middleware between the database <br>
+    and the user through the browser. A principle benefit of a CMS is that it allows non-technical <br>
+    people to make changes to their web site without having technical knowledge.</p>
+    
+</body>
+</html>
```

Comment:
Amazing job on the challenges overall. You write clean code, it's very easy to read. 
Just make sure you validate EVERYTHING. It seems excessive, but it takes just a few seconds, and it will help you avoid any errors.